Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -1,594 +1,354 @@
-/* Style applied to the BocList. */
-
 div.bocList
```

Comment:
the layout for boclist as grid is broken. either because of the grid mode or because a boclist without vertical scrolling doesn't yet work. please check both scenarios.